scripts/cnode-helper-scripts/gLiveView.sh

```diff
@@ -150,14 +149,13 @@ if [[ "${NO_INTERNET_MODE}" == "N" ]]; then
       fi
     else
       mv "${PARENT}"/env.tmp "${PARENT}"/env
-      echo -e "Common env file downloaded: ${PARENT}/env"
-      echo -e "This is a mandatory prerequisite, please set variables accordingly in User Variables section in the env file and restart Guild LiveView\n"
-      exit 0
+      myExit 0 "Common env file downloaded: ${PARENT}/env\nThis is a mandatory prerequisite, please set variables accordingly in User Variables section in the env file and restart Guild LiveView\n"
```

Comment:
Minor wording since we already retain user variables
```suggestion
      myExit 0 "Common env file downloaded: ${PARENT}/env\nUser variables will be retained, but please perform a sanity check in User Variables section of env file and restart Guild LiveView\n"
```

Reply:
This message is only output if this check fails: `if [[ -f "${PARENT}"/env ]]; then`
Meaning that no previous env file is found, ie gLiveView.sh script has been downloaded and executed without env being present in the same dir. On update with retained User Variables for env file, we output the message: `echo -e "\nUpdate to env file applied successfully!"`. Though actually pointless in the end if we don't add a waitForInput() type of code after as code will continue and clear screen without the user ever seeing the output.

Reply:
ah right, agree